client/components/AvailabilityGrid.js

```diff
@@ -5,10 +5,13 @@ import moment from 'moment';
 import autobind from 'autobind-decorator';
 import fetch from 'isomorphic-fetch';
 import { checkStatus } from '../util/fetch.util';
-import { getHours, getMinutes } from '../util/time-format';
+import { getHours, getMinutes, addZero, removeZero  } from '../util/time-format';
```

Comment:
WE not using now. But in the future who knows ? So at least make the code at AvaiabilityGrid or any other long - more the 100 lines - file easier to read and maintain.